packages/expect/src/utils.ts

```diff
@@ -133,6 +133,9 @@ export const getObjectSubset = (object: any, subset: any): any => {
   return object;
 };
 
+export const getObjectEntries = (object: any): Array<any> =>
```

Comment:
Add a comment to remove this when we drop node 6?

Reply:
Does it work type wise to do `export const getObjectEntries: typeof Object.entries = object =>`?